src/families/polkadot/js-getTransactionStatus.js

```diff
@@ -69,7 +71,24 @@ const getSendTransactionStatus = async (
     errors.amount = new AmountRequired();
   }
 
-  if (totalSpent.gt(a.spendableBalance)) {
+  const existentialDeposit = getExistentialDeposit(a);
```

Comment:
Same comment: would name this `minimumBalance` maybe, since existential deposit has a precise meaning.